report_custom_filename/README.rst

```diff
@@ -8,12 +8,6 @@ Configuration
 
 To configure this module, open the report whose filename you want to change and fill in the `Download filename` field. This field is evaluated as jinja2 template with `objects` being a list of browse records of the records to print, and `o` the first record.
```

Comment:
Can you take the occasion to put here an example of a simple jinja2 example (for instance `${o.name + '_report'}`)?